lib/internal/vm/module.js

```diff
@@ -58,6 +58,15 @@ const kContext = Symbol('kContext');
 const kPerContextModuleId = Symbol('kPerContextModuleId');
 const kLink = Symbol('kLink');
 
+// Helper caried over from js2c_macros: exits process and prints debug message
+// iff node_debug_lib flag is set.
+function CHECK(x) {
+  if (process.config.variables.node_debug_lib === false) return;
```

Comment:
It's probably better to use `process.features.debug` here.

Reply:
done 👍